platform/ui/src/components/StudyListPagination/StudyListPagination.js

```diff
@@ -18,12 +19,12 @@ const StudyListPagination = ({
       <div className="container m-auto relative px-8">
         <div className="flex justify-between">
           <div className="flex items-center">
-            <div className="relative mr-3">
+            <div className="PageSelect relative mr-3">
               <select
                 value={perPage}
-                className="block appearance-none w-full bg-transparent border border-common-active text-white text-base px-2 pr-4 rounded leading-tight focus:outline-none h-8"
+                className={`block appearance-none w-full border border-common-active ${isOS('Win') ? 'bg-black' : 'bg-transparent'} text-white text-base px-2 pr-4 rounded leading-tight focus:outline-none h-8`}
```

Comment:
This issue was a different behavior for Chrome versus FireFox issue, not operating system:

https://precisionmetrics.atlassian.net/browse/OHIF-298

I'd prefer we use appropriate vendor prefixes, or vendor support styles instead of using JS to detect. There should be a pure CSS fix. If there's not, I'd rather we fallback to whatever fancy `select` component we're using which is likely a bit more battle hardened.